csvkit/cli.py

```diff
@@ -134,6 +135,8 @@ def run(self):
         if 'f' not in self.override_flags:
             self.input_file = self._open_input_file(self.args.input_path)
 
+        AddBOM.run(self.output_file, self.args)
```

Comment:
Just inline the 2 lines of code here, instead of creating a new 54-line file:

```python
if getattr(self.args, 'add_bom', False):
    self.output_file.buffer.write(BOM_UTF8)
```

And of course do `from codecs import BOM_UTF8` with the other imports.